scripts/update-rules.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const rules = require('../lib/rules');
+const { rules: recommendedRules } = require('../lib/config/recommended');
+const { rules: octaneRules } = require('../lib/config/octane');
+const { rules: stylisticRules } = require('../lib/config/stylistic');
+
+const pathReadme = path.resolve(__dirname, '../README.md');
+const readmeContent = fs.readFileSync(pathReadme, 'utf8');
+const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/;
+
+// Config/preset emojis.
+const EMOJI_STAR = ':white_check_mark:';
+const EMOJI_OCTANE = ':car:';
+const EMOJI_STYLISTIC = ':dress:';
+
+// Generate rule table contents.
+let rulesTableContent = Object.keys(rules)
```

Comment:
Is there a reason this isn't `const` (and that it isn't being enforced)?

Reply:
Fixed. I also prefer to use const. But here's why it's not enforced inside this package: https://github.com/ember-template-lint/ember-template-lint/pull/830#issuecomment-534148039